src/ui/public/doc_table/doc_table.js

```diff
@@ -55,6 +55,12 @@ uiModules.get('kibana')
         $scope.limit += 50;
       };
 
+      $el.attr('has-render-count', 'true');
+      $el.attr('render-count', '0');
+      $el.on('renderComplete', () => {
+        $el.attr('render-count', parseInt($el.attr('render-count'), 10) + 1);
```

Comment:
maybe a dumb micro-optimization, but reading this off the DOM every time isn't free. Why not cache the value in the directive?